README.md

```diff
@@ -1,6 +1,5 @@
 # Czg-Keramony
```

Comment:
Please fix the typo 🙏 
`Czg-keramony` -> `CZG-Keremony`

Reply:
Fixed 1286aea83986fa66360b20fee7a44472af5dc0b1